src/lxc/lxc_user_nic.c

```diff
@@ -128,13 +241,48 @@ static int get_alloted(char *me, char *intype, char *link)
 			continue;
 		if (strcmp(link, br) != 0)
 			continue;
-		free(line);
-		fclose(fin);
-		return n;
+
+		append_alloted(alloted, me, n);
+		count += n;
+		break; // found the user with the appropriate settings, therefore finish the search.
+		// what to do if there are more than one applicable lines? not specified in the docs.
+		// since getline is implemented with realloc, we don't need to free line until exiting func.
```

Comment:
Code style prefers /* */ for multi-line comments and those should be placed ahead of the code.